### PR TITLE
[Doc] add refresh mv with sync mode doc

### PR DESCRIPTION
**Pull request by ABingHuang**
## What type of PR is this:
- [ ] BugFix
- [ ] Feature
- [ ] Enhancement
- [ ] Refactor
- [ ] UT
- [ ] Doc
- [ ] Tool

## Checklist:
- [ ] I have added test cases for my bug fix or my new feature
- [ ] This pr will affect users' behaviors
- [ ] This pr needs user documentation (for new or modified features or behaviors)
  - [ ] I have added documentation for my new feature or new function

## Bugfix cherry-pick branch check:
- [ ] I have checked the version labels which the pr will be auto-backported to the target branch
  - [ ] 3.1
  - [ ] 3.0
  - [ ] 2.5
  - [ ] 2.4

### CODE REVIEW

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]


:heart_eyes: **pass** : 0 / 0 (0%)

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]


:heart_eyes: **pass** : 0 / 0 (0%)

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]


:heart_eyes: **pass** : 0 / 0 (0%)

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]


:heart_eyes: **pass** : 0 / 0 (0%)

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]


:heart_eyes: **pass** : 0 / 0 (0%)

File: docs/sql-reference/sql-statements/data-manipulation/REFRESH MATERIALIZED VIEW.md [hunk 1]
```diff
@@ -12,36 +12,52 @@ Manually refresh a specific asynchronous materialized view or partitions within.
...
-Example 2: Manually refresh certain partitions of a specific materialized view
+Example 2: Manually refresh certain partitions of a specific materialized view.
 
 ```Plain
-REFRESH MATERIALIZED VIEW mv 
+REFRESH MATERIALIZED VIEW lo_mv1 
 PARTITION START ("2020-02-01") END ("2020-03-01");
 ```
+
+Example 3: Manually refresh certain partitions of a specific materialized view by force.
```
Comment (evelynzhaojie) [inline docs/sql-reference/sql-statements/data-manipulation/REFRESH MATERIALIZED VIEW.md:52]: by force 不太合适，感觉常用的意思是”通过武力解决，physical power“，建议：Forcibly refresh certain partitions of a specific materialized view.

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]


:heart_eyes: **pass** : 0 / 0 (0%)

**Review by amber-create — APPROVED** [review]

**Review by evelynzhaojie — APPROVED** [review]

**Comment by wanpengfei-git** [comment]
@Mergifyio backport branch-3.1

**Comment by wanpengfei-git** [comment]
@Mergifyio backport branch-3.0

**Comment by wanpengfei-git** [comment]
@Mergifyio backport branch-2.5

**Pull request merged**